.github/workflows/build_and_test_workflow.yml

```diff
@@ -346,7 +346,7 @@ jobs:
 
       - name: Set OpenSearch URL
         run: |
-          echo "OPENSEARCH_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ env.VERSION }}/latest/linux/x64/tar/dist/opensearch/opensearch-${{ env.VERSION }}-linux-x64.tar.gz" >> $GITHUB_ENV
+          echo "OPENSEARCH_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ env.VERSION }}/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-${{ env.VERSION }}-linux-x64.tar.gz" >> $GITHUB_ENV
```

Comment:
I see the min is failing a different test now. The min for OpenSearch Dashboards is passed with a `--release` which compiles down our files to JS directly so any `dev` mode only stuff won't work on the `min`. I do not remember if the `min` is the same as the `snapshot` version for OpenSearch.

Quickly looking the link provided by the snapshot doesn't have the same metadata as the one on the min when I checked locally (but that isn't proof that they are different just suspect).

Do you have a reference to this pass on the 2.x branch that I can peek at?